.style.yapf

```diff
@@ -3,5 +3,5 @@ based_on_style=pep8
 spaces_before_comment=4
 split_before_logical_operator=True
 indent_width=4
-column_limit=140
```

Comment:
any reasons for 130?

Reply:
No I can leave it at 140 if you want. I'm not sure what the best thing is here.